docs/source-fabric/guide/checkpoint/index.rst

```diff
@@ -0,0 +1,32 @@
+:orphan:
```

Comment:
if we list it in Glossary, would be easier...

Reply:
It is linked in the Glossary:

```
.. displayitem::
    :header: Checkpoint
    :button_link: ../guide/checkpoint/index.html
    :col_css: col-md-4
```

Reply:
yes, but this is HTML not the source document that is why you need to add this tag `:orphan:` so I may suggest having both... so when it is in any tree it is a bit harder to lose it :)

Reply:
I added it now